datafusion/functions-aggregate-common/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//! Aggregate Function Common packages for [DataFusion]
+//! This package could be used to build for 3rd party aggregate function
+//!
+//! [DataFusion]: <https://crates.io/crates/datafusion>
```

Comment:
```suggestion
//! Common Aggregate functionality for [DataFusion]
//!
//! This crate contains traits and utilities commonly used to implement aggregate functions
//! They are kept in their own crate to avoid physical expressions depending on logical expressions.
//!
//! [DataFusion]: <https://crates.io/crates/datafusion>
```